x-pack/plugins/apm/server/routes/alerts/rule_types/docs/params_property_apm_error_count.yaml

```diff
@@ -0,0 +1,35 @@
+# @kbn-doc-linker partial
+
+params_property_apm_error_count:
+  required:
+    - windowSize
+    - windowUnit
+    - threshold
+    - environment
+  properties:
+    serviceName:
+      type: string
+      description: The service name from APM
+    windowSize:
+      type: number
+      description: The window size
+      example: 6
+    windowUnit:
+      type: string
+      description: The window size unit
+      enum:
+        - "m"
+        - "h"
+        - "d"
+    environment:
+      type: string
+      description: The environment from APM
+    threshold:
+      type: number
+      description: The error count threshold value
+    groupBy:
```

Comment:
Here are the available: `
["service.name", "service.environment", "transaction.name", "error.grouping_key"]`

with `service.name", "service.environment` applied by default

Reply:
Updated with https://github.com/elastic/kibana/pull/158676/commits/5cf8b5e1963c1543378ffd7e0b48704b052bc86b

Reply:
Updated with https://github.com/elastic/kibana/pull/158676/commits/5cf8b5e1963c1543378ffd7e0b48704b052bc86b